recipes/pscript/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pscript" %}
+{% set version = "0.5.3" %}
+{% set sha256 = "cabd070f47329d5e7b23cb479cced5da38492262d742bca9861383a06d71522c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - pscript
+
+about:
+  home: http://pscript.readthedocs.io
+  doc_url: http://pscript.readthedocs.io
+  dev_url: https://github.com/flexxui/pscript
+  summary: library for transpiling Python code to JavaScript.
+  license: BSD 2-Clause
```

Comment:
Would be good to package the license file. Have raised issue ( https://github.com/flexxui/pscript/issues/16 ) about getting this into the PyPI package.

Reply:
note: Done below.